frontend/ui/data/keyboardlayouts/keypopup/en_popup.lua

```diff
@@ -186,9 +189,10 @@ return {
         north = "ï",
         northeast = "í",
         northwest = "ì",
-        east = "î",
+        east = "î", -- i with circumflex (Romanian, French)
```

Comment:
```suggestion
        east = "î", -- i with circumflex (e.g., French, Romanian)
```
E.g. because I think it would be a bit tiresome to list the whole shebang of Afrikaans, French, Kurdish, Italian, Romanian, Turkish, Welsh, Walloon, and possibly more. The order for alphabetical reasons (easier to scan at a glance).

Reply:
Ok, I ordered them alphabetically. :)